File: GDocs.Site/src/app/assinatura/models/assinatura.model.ts
import { AssinaturaInformacoesModel, AssinaturaUploadResponseModel, PassoAssinaturaUsuariosAdAdcionadosPaiModel } from ".";

export class AssinaturaModel {
  constructor(public arquivos: AssinaturaUploadResponseModel[], public informacoes: AssinaturaInformacoesModel, public passos: PassoAssinaturaUsuariosAdAdcionadosPaiModel) { }
}

File: GDocs.Api/src/Infra/CrossCutting/Models/AssinaturaModel.cs
using System.Collections.Generic;

namespace ICE.GDocs.Infra.CrossCutting.Models
{
    public class AssinaturaModel
    {
        public AssinaturaModel()
        {
            Passos = new AssinaturaPassoModel();
        }

        public AssinaturaInformacoesModel Informacoes { get; set; }
        public AssinaturaPassoModel Passos { get; set; }        
        public IEnumerable<AssinaturaArquivoModel> Arquivos { get; set; }
    }
}
